Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxy.cpp

```diff
@@ -81,15 +81,15 @@ RefPtr<RemoteGraphicsContextGLProxy> RemoteGraphicsContextGLProxy::create(IPC::C
     unsigned connectionBufferSizeLog2 = defaultConnectionBufferSizeLog2;
     if (attributes.failContextCreationForTesting == WebCore::GraphicsContextGLAttributes::SimulatedCreationFailure::IPCBufferOOM)
         connectionBufferSizeLog2 = 50; // Expect this to fail.
-    auto [clientConnection, serverConnectionHandle] = IPC::StreamClientConnection::create(connectionBufferSizeLog2);
-    if (!clientConnection)
+    auto connectionPair = IPC::StreamClientConnection::create(connectionBufferSizeLog2);
+    if (!connectionPair || !connectionPair->streamConnection)
         return nullptr;
-    auto instance = platformCreate(connection, clientConnection.releaseNonNull(), attributes
+    auto instance = platformCreate(connection, connectionPair->streamConnection.releaseNonNull(), attributes
```

Comment:
in the if continuation you could just deconstruct like the previous code:
```
auto [clientConnectin, serverConnectionHandle] = *connectionPair;
```

This would avoid the long names.

Reply:
Just a heads up that I didn't see that compiling but the following it was happy with.

```cpp
auto [clientConnectin, serverConnectionHandle] = WTFMove(*connectionPair);
```